server/lib/tuist/github/client.ex

```diff
@@ -41,6 +41,41 @@ defmodule Tuist.GitHub.Client do
   `repository_full_handle` is necessary as to interact with the user endpoint,
   we need to be authenticated with the GitHub app installation token associated with a specific repository.
   """
+  def get_installation_repositories(installation_id) do
+    case App.get_installation_token(installation_id) do
+      {:ok, %{token: token}} ->
+        url = "https://api.github.com/installation/repositories"
+
+        case Req.get(
```

Comment:
Should we paginate here? I'd expect some organizations to have thousands of repositories.